src/components/patterns/image-card.tsx

```diff
@@ -33,7 +33,7 @@ const ImageCard = ({imageUrl, imageAlt, videoUrl, isArticle, children, hasBgColo
   return (
     <CardWrapper
       {...props}
-      className={twMerge("centered w-full rounded-[25px] lg:max-w-[980px]", hasBgColor ? "bg-transparent" : "bg-fog-light", props.className)}
+      className={twMerge("centered w-full rounded-[25px] lg:max-w-[980px]", hasBgColor ? "flex h-full flex-col justify-center bg-transparent" : "bg-fog-light", props.className)}
```

Comment:
are the flex classes necessary? maybe they could be applied all the time? 
`clsx` could be more clear for what classes are to be active

Reply:
Actually, having those flex classes really causes chaos. using the `/style-samples` page, it causes the card to push everything down and overlap other items.

Reply:
I used the flex classes to achieve this center alignment but I'll look into alternative styling options that would work well in the masonary grid format shown on the `/style-samples` page 🤔  
![image](https://github.com/SU-SWS/summer-nextjs/assets/34019925/ccb24dc5-bdc6-48f5-815e-34a33a21e258)

Reply:
Hmm. I added a padding top instead as a workaround to having it slightly offset. Should we have equal spacing (top and bottom) instead? I di wonder if there are other transparent card combo scenarios that we should account for outside of what we have designed.

Any thoughts? 🤔

Reply:
I don't know that there's going to be a good solution for that. I think the padding will be sufficient for now. If they need changes, we can come back to it later.

Reply:
🤔 Alternatively, we could provide spacer options in the behaviors so that the content author can decide if they would like top and/or bottom padding. But that could also introduce unnecessary complexity.